SConscript.firmware

```diff
@@ -85,6 +86,31 @@ SOURCE_MOD += [
     'vendor/trezor-crypto/sha3.c',
 ]
 
+# libsecp256k1-zkp
+CPPPATH_MOD += [
+    'vendor/secp256k1-zkp/',
```

Comment:
Do we need to have this line in the list? (`'vendor/secp256k1-zkp/'`)

Reply:
Yes, that's how library is built. Otherwise you'll get 
```
vendor/secp256k1-zkp/src/secp256k1.c:7:10: fatal error: include/secp256k1.h: No such file or directory #include "include/secp256k1.h"
```

Is this problematic?

Reply:
It's not. OK